src/dateutils.ts

```diff
@@ -20,7 +20,30 @@ export function sameDate(a?: XDate, b?: XDate) {
     return false;
   } else {
     return a?.getFullYear() === b?.getFullYear() && a?.getMonth() === b?.getMonth() && a?.getDate() === b?.getDate();
-  } 
+  }
+}
+
+export function onSameDAteRange({
```

Comment:
The "A" on Date is capitalized... typo I guess

Reply:
fixed